### PR TITLE
Bug fixes and minor improvements

### PR DESCRIPTION
**Pull request by carina-akaia**

### CODE REVIEW

**Comment by carina-akaia** [comment]
@race-of-sloths

**Comment by codingshot** [comment]
@race-of-sloths score [1]

**Comment by race-of-sloths** [comment]
### 🏆 Score recorded! 
It looks like @codingshot provided a score of [1], but based on our Fibonacci scoring rules, we've adjusted it to 0.
> [!TIP]
We accept only the following Fibonacci numbers [0, 1, 2, 3, 5, 8, 13] for scoring

File: src/modules/core/components/Nav.tsx [hunk 1]
```diff
@@ -17,7 +18,12 @@ const links = [
   { label: "Pots", url: routesPath.POTS, disabled: false },
   { label: "Feed", url: routesPath.FEED, disabled: false },
   // { label: "Donors", url: routesPath.DONORS, disabled: false },
-  // { label: "Lists", url: routesPath.LIST, disabled: false },
+
+  {
+    label: "Lists",
+    url: routesPath.LIST,
+    ...(DEBUG ? {} : { disabled: true }),
+  },
```
Comment (Ebube111) [inline src/modules/core/components/Nav.tsx:26]: Don't show the list tab just yet, @Prometheo  is has not deployed his changes to mainnet, most things are breaking

File: src/modules/donation/hooks/forms.ts [hunk 2]
```diff
@@ -187,6 +190,12 @@ export const useDonationForm = ({
     isSingleProjectDonation,
   ]);
 
+  console.log(values.groupAllocationPlan);
+
+  console.table({ hasChanges, isValid: self.formState.isValid });
+
+  console.log(JSON.stringify(self.formState, null, 2));
```
Comment (Ebube111) [inline src/modules/donation/hooks/forms.ts:197]: do you need cleanup?

**Review by Ebube111 — CHANGES_REQUESTED** [review]
Code looks good. but we need to hide the List tab from the nav. it's not ready till the contract changes are updated

File: src/modules/core/components/Nav.tsx (hunk 1 above)
Comment (Ebube111) [reply src/modules/core/components/Nav.tsx:26]: or does the disabled prop hide it?

**Review by Ebube111 — APPROVED** [review]

File: src/modules/core/components/Nav.tsx (hunk 1 above)
Comment (carina-akaia) [reply src/modules/core/components/Nav.tsx:26]: No, but it should be non-clickable for regular users, do we really have to comment it out?

File: src/modules/donation/hooks/forms.ts (hunk 2 above)
Comment (carina-akaia) [reply src/modules/donation/hooks/forms.ts:197]: This time I'll need it in the next PR I'll open soon enough

Comment (Ebube111) [reply src/modules/donation/hooks/forms.ts:197]: Noted

File: src/modules/core/components/Nav.tsx (hunk 1 above)
Comment (Ebube111) [reply src/modules/core/components/Nav.tsx:26]: Alright sounds good

**Pull request merged**